### DISCUSSION TITLE
[bug] Pinned discussion box having unwanted scrollbars

### DISCUSSION THREAD

**Discussion by jonathanfallon** [community/community#3014] [Discussions] [answered]
OS : Ubuntu 20.10
Browser : Brave Version 1.16.72 Chromium: 86.0.4240.183 (Official Build) (64-bit)

![image](https://user-images.githubusercontent.com/1656255/102497976-a77ade80-4079-11eb-9038-c64893fae22c.png)

Changing window size doesn't fix the issue, even after a reload.

**Comment by bosborn** [comment]
Duplicate of #2995?

**Comment by becca** [comment, marked as answer]
This should now be fixed

**Reply by jonathanfallon** [reply]
Fixed. Thanks a lot !
